packages/admin/admin/src/snackbar/SnackbarProvider.tsx

```diff
@@ -60,6 +61,7 @@ export const SnackbarProvider: React.FunctionComponent = ({ children }) => {
             {children}
             {snackbar !== undefined &&
                 React.cloneElement<SnackbarProps>(snackbar, {
+                    key: UUID.v4(),
```

Comment:
This creates a new key on every render of `SnackbarProvider`. I' not sure that this works correctly

Reply:
I believe there shouldn't be any rerenders because the snackbar is only updated when `showSnackbar()` is called.

But to be sure I now added a `key` state that is only updated when `showSnackbar()` is called.